Make the CPUs question a bit less technical (#115)
rust-lang/surveys: surveys/2021-annual-survey/questions.md

> specific answer appears frequently, we can add it to future surveys to reduce
> the amount of work needed to process responses.

> ### On the primary machine you compile Rust code on, how many CPU do you have?

Reviewer: ```suggestion
### On the primary machine you compile Rust code on, how many CPUs are there?
```
Even if you decide to keep the original wording, we should use the plural of CPUs here.
